roles/setup/handlers/main.yml

```diff
@@ -5,7 +5,9 @@
     state: restarted
     enabled: true
   register: _restart
-  when: not _started.changed # don't restart if we recently started
+  when:
+    - not _started.changed # don't restart if we recently started
+    - receptor_install_method in ['release', 'local', 'package']
```

Comment:
what reload is not enough for the containerized deployment? Should we have another handler for that case? Looks like a good opportunity to use [listen](https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_handlers.html#naming-handlers)

Reply:
The systemd reload in the user context needs the variable `XDG_RUNTIME_DIR` defined in order to run successfully.

If I execute via handler, I cant pass this env variable in a reliable way that I know of currently, which is why I do not reload the systemd via handler in this case, but rather right after the the task pod is running